src/GitHub/Event/GitHubStatus.php

```diff
@@ -21,14 +21,30 @@
  */
 final class GitHubStatus extends AbstractGitHubEvent
 {
+    /**
+     * Patterns for build contexts we're interested in.
+     *
+     * @var string[]
+     */
     private const CONTEXT_PATTERNS = [
+        '#coveralls#',
         '#^github#',
         '#travis-ci#',
-        '#coveralls#',
     ];
```

Comment:
Does this include DSO check? If not, do we need notifications about it?

Reply:
DCO check is a github action, IIRC, but I'll need to verify. That should be handled in a separate PR, though.